segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/stat/SegmentStatsRecorderImpl.java

```diff
@@ -176,7 +176,9 @@ private SegmentWriteContext getWriteContext(String streamSegmentName) {
     public void createSegment(String streamSegmentName, byte type, int targetRate, Duration elapsed) {
         this.createStreamSegment.reportSuccessEvent(elapsed);
         SegmentAggregates sa = SegmentAggregates.forPolicy(ScaleType.fromValue(type), targetRate);
-        cache.put(streamSegmentName, new SegmentWriteContext(streamSegmentName, sa));
+        if (!NameUtils.isTransactionSegment(streamSegmentName)) {
+            cache.put(streamSegmentName, new SegmentWriteContext(streamSegmentName, sa));
```

Comment:
@shiveshr this change will effectively make [SegmentStatsRecorderImpl.policyUpdate()](https://github.com/pravega/pravega/blob/master/segmentstore/server/host/src/main/java/io/pravega/segmentstore/server/host/stat/SegmentStatsRecorderImpl.java#L214) a no-op for transaction segments. Can you please confirm that there is no side effect of this one some other logic?

Reply:
this should be inconsequential for txn segments.. this is because there is no policy update on a transaction segment anyway as the segment policy is about scaling and txn segments never scale.